joomla/joomla-cms
# [3.x] TinyMCE make use of the core textfilter API for new sites

### Summary of Changes

TinyMCE make use of the core textfilter API for new sites

### Testing Instructions

- Install the new package [(from the package step)](https://ci.joomla.org/artifacts/joomla/joomla-cms/staging/28721/downloads/31369/) and check that the default settings for new installs are changed.
- install the update package [(from the package step)](https://ci.joomla.org/artifacts/joomla/joomla-cms/staging/28721/downloads/31369/) and make sure no changes are made for upgrades

### Expected result

The TInyMCE uses the Core textFilter API

### Actual result

The TinyMCE has its own textfilters that just confuses people new to joomla and does not help to protect as everyone is free to change the editor.

### Documentation Changes Required

none

cc @SniperSister  @widmann-it @coolcat-creations
